docs/contribute/code_review.md

```diff
@@ -0,0 +1,68 @@
+# Code Review
+
+MDC Web team members review every single PR before it is merged into the code
+base. We aim for a quick review process, but without sacrificing good component
+APIs and code maintenance.
+
+## PR Assignments
+
+We currently have six team members, and we normally have bandwidth to respond to
+two PRs each a day (thats twelve PRs total). We do NOT have an SLA to respond
+to PRs within some timeframe. If there are lots of PRs to review, then we
+simply might not get to yours for a couple days.
+
+Once a MDC Web team member has started reviewing a PR, they will be the only
+point of contact until the PR is closed. The reviewer should self-assign so that
+other team members don't review the same code.
+
+If an MDC Web team member requests changes, and the contributor hasn't responded
+in more than two weeks, the reviewer might close the PR. This saves our sanity
+of having too many open pull requests.
+
+## Checklist
+
+### Check The Obvious
+
+*  Do tests pass?
+*  Does the demo work?
+
+If either of these don’t work, send that review first before diving into
```

Comment:
~If either~ If any

^ Provided you add the code coverage bullet

Reply:
The language used is a little foggy: "If either of these don’t work, send that review first before diving into API design"

Maybe consider something like: "If any of these checks fail, request changes from the developer before reviewing the contents of the pull request."